utils/slackhelper.py

```diff
@@ -0,0 +1,87 @@
+from slack import WebClient, errors
+
+
+class SlackHelper:
+    """Wraps official Slack's client functions and add additional features:
+        - search in users list
+        - extract only one field
+    """
+
+    def __init__(self, token: str):
+        """Init official client.
+
+            token (str):
+                is the token for Api calls authentication.
+                https://api.slack.com/start/overview
+        """
+        self.slack_client = WebClient(token=token)
+
+    def get_users(self, search: str = None, field: str = None) -> list:
+        """Calls Slack's endpoint `users.list`.
+
+            search (str):
+                if is not None the function returns only slack's users
+                who match string at least one field.
+            field (str):
+                if is not None the function will return a list containing
+                only specified field.
+
+            Returns list of slack's users or
```

Comment:
```suggestion
            Returns a list of slack's users or
```

Non sono nemmeno sicuro che vada usato il genitivo sassone (@tommyblue ?)